### PR TITLE
Support for running tests from setup.py

### PR DESCRIPTION
**Pull request by ashfire908**
For #42.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -42,5 +42,7 @@
   keywords="faunadb fauna",
   packages=["faunadb"],
   install_requires=requires,
-  extra_requires=extra_requires
+  extras_require=extra_requires,
```
Comment (andy-faunadb) [inline setup.py]: Should probably name both `extras_require` then.

Comment (ashfire908) [inline setup.py]: Yeah, lemme fix that

**Comment by andy-faunadb** [comment]
So, how do you run tests from setup.py?

**Comment by ashfire908** [comment]
`python setup.py test`.

**Comment by andy-faunadb** [comment]
LGTM

**Pull request merged**